TransIT.BLL/Services/UserService.cs

```diff
@@ -0,0 +1,128 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TransIT.BLL.Security.Hashers;
+using TransIT.DAL.Models.Entities;
+using TransIT.DAL.Repositories.InterfacesRepositories;
+using TransIT.DAL.UnitOfWork;
+
+namespace TransIT.BLL.Services
+{
+    /// <summary>
+    /// User model CRUD
+    /// </summary>
+    public class UserService : IUserService
+    {
+        /// <summary>
+        /// Saves changes
+        /// </summary>
+        private readonly IUnitOfWork _uow;
+        /// <summary>
+        /// User repo
+        /// </summary>
+        private readonly IUserRepository _userRepository;
+        /// <summary>
+        /// Roles repo
+        /// </summary>
+        private readonly IRoleRepository _roleRepository;
```

Comment:
We don't need repositories here because UnitOfWork already contains them

Reply:
UnitOfWork, but not IUnitOfWork, I think we should reference on abstraction. Maybe IUnitOfWork should have them.

Reply:
Of course it should